### PR TITLE
add mobile-menu hamburger toggle show

### PR DESCRIPTION
**Pull request by fickryiman**
add mobile menu 
- toggle show and hide for the menu
- add javascript function to change style display

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

⚠️ You have submitted a project, that has no visible contribution from your pair programming partner. Kindly check [Work and Submission mode](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode).
Screenshot 2023-02-08 at 9 56 02 PM

⚠️ Please add a Pull Request description including changes made on this branch. You can this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects) to learn more.
Screenshot 2023-03-06 at 7 36 56 PM



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by didierganthier — COMMENTED** [review]
Hi Team,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- You have submitted a project with an empty PR description. The requirement states that the Pull Request should have a short summary describing introduced changes in more detail. This is what you have 👇🏽

Screenshot 2023-03-06 at 23 40 02


Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

File: js/mobile-menu.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  mobileNav.style.display = 'unset';
+}
+
+function hideMobileMenu() {
+  mobileNav.style.display = 'none';
+  defaultNav.style.display = 'unset';
+}
+
+hamburger.addEventListener('click', displayMobileMenu);
+xButton.addEventListener('click', hideMobileMenu);
```
Comment (didierganthier) [inline js/mobile-menu.js:17]: Well done overall 💯 but one of the requirements is that when the user clicks (or taps) any option of the mobile menu, the mobile menu should disappear. Yours is not disappearing for now. 😶‍🌫️

**Review by didierganthier — CHANGES_REQUESTED** [review]
Hi Team,

🎉 Good job so far! You're making great progress on this project, but there are still a few areas that need attention before we can move on to the next step.



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Menu working ✔️ 

## Required Changes 🛠
- Please review the comments under the review for specific changes that still need to be made.

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code.

🚀 Keep pushing forward, you're almost there! If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Cheers and happy coding! 👏👏👏

File: js/mobile-menu.js (hunk 1 above)
Comment (fickryiman) [reply js/mobile-menu.js:17]: add an event listener to hide the mobile menu when the user clicks the option

Comment (fickryiman) [reply js/mobile-menu.js:17]: hide the mobile menu when the user clicks DONE.

**Review by addod19 — APPROVED** [review]
### Hi @fickryiman and @KUBAHO3, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [x] Nice job fixing the disappearing of the menu list :rocket: 

### Optional
- [x] It would be better to include the license header in our README file as well as the MIT license file :man_technologist: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**